### DISCUSSION TITLE
BYU-IDAHO WDD231 Project

### DISCUSSION THREAD

**Discussion by d9linkx** [community/community#143245] [Accessibility]
### Select Topic Area

General

### GitHub Feature Area

Projects

### Body

Hi instructor, here's the video of my project, WDD231:

Uploading freecompress-Screen Recording 2024-11-01 at 04.03.23.mp4…

**Comment by d9linkx** [comment]
Uploading freecompress-Screen Recording 2024-11-01 at 04.03

https://github.com/user-attachments/assets/6666c58b-08ee-4337-82af-831a6a2cde57

.23.mp4…

**Comment by ebndev** [comment]
Hi @d9linkx, thanks for being a part of the GitHub Community!

GitHub Community Discussions is a public forum and focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. You may have meant to post in your class's Discussions board or another platform.

We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular post.

Best of luck!
